pkgs/servers/firebird/4.nix

```diff
@@ -0,0 +1,59 @@
+{ autoreconfHook
+, fetchFromGitHub
+, icu
+, lib
+, libedit
+, libtomcrypt
+, libtommath
+, stdenv
+, unzip
+, zlib
+
+, superServer ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "firebird";
+  version = "4.0.2";
+
+  src = fetchFromGitHub {
+    owner = "FirebirdSQL";
+    repo = "firebird";
+    rev = "v${version}";
+    sha256 = "sha256-hddW/cozboGw693q4k5f4+x9ccQFWFytXPUaBVkFnL4=";
+  };
+
+  configureFlags = [
+    "--with-system-editline"
+  ] ++ (lib.optional superServer "--enable-superserver");
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath [ icu ];
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [
+    icu
+    libedit
+    libtomcrypt
+    libtommath
+    unzip
+    zlib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -r gen/Release/firebird/* $out
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "SQL relational database management system";
+    downloadPage = "https://github.com/FirebirdSQL/firebird/";
+    homepage = "https://firebirdsql.org/";
+    changelog = "https://github.com/FirebirdSQL/firebird/blob/master/CHANGELOG.md";
```

Comment:
This should point at the changelog of the corresponding tag.

Reply:
Done